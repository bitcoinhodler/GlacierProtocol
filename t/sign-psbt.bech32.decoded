{
  "txid": "ea99066c9eea823ea6abf026b0c5eb79aaf86f8b5c57bfa98ac602dc0a399094",
  "hash": "a2b7ad0fed898975c0b46f3114e63d7f95183b0ddbba30208b411e5865804f40",
  "version": 2,
  "size": 405,
  "vsize": 189,
  "weight": 756,
  "locktime": 0,
  "vin": [
    {
      "txid": "47448c1c49fadbf4745f36d4bb7bc8240e7900bbf17e797e30d0bba83dc374b6",
      "vout": 1,
      "scriptSig": {
        "asm": "002065c863c750331cc03efb4daca217e7925cff87f5e9ca9927564234be0d6278e8",
        "hex": "22002065c863c750331cc03efb4daca217e7925cff87f5e9ca9927564234be0d6278e8"
      },
      "txinwitness": [
        "",
        "304402203d122d6cb5c2ab9a937ed9a8d75ba97803d2833034a687505623f531211f1fcf022057eb3975c586159f5fb57b7103cbe831fecc357843983689c1c7e73afdbd870401",
        "3044022063b978b9c2a41562adf242c660afc8ea5834a04a9a117add258f7c3624b2dbe902207896204b1345094b7a3a17bf4a642bff3f936448cb11ce2a668a56a7c3277d3d01",
        "522103d14ddcfb6817f5579695bbb3eb3e185887bf2942b031e6f716343b8fe7e9e8e221028fcd46f8614b2cbf318096968242a1e22bcfb6d8f2b6dc939c8c27c347b2937b210315acb550120f4cdcb460d5c49080ab508ca4ccd8dedecac22feeac8cd017d4c121022b063ee2f22f9e1982c140fe778672c683111a796dcccdbd47bb65e3d608a98354ae"
      ],
      "sequence": 4294967295
    }
  ],
  "vout": [
    {
      "value": 0.99889303,
      "n": 0,
      "scriptPubKey": {
        "asm": "0 ce62ba41a5fc35dff5d97534004f984d6a89797d",
        "desc": "addr(bcrt1qee3t5sd9ls6alawew56qqnucf44gj7tajk3232)#rfwfrq4n",
        "hex": "0014ce62ba41a5fc35dff5d97534004f984d6a89797d",
        "address": "bcrt1qee3t5sd9ls6alawew56qqnucf44gj7tajk3232",
        "type": "witness_v0_keyhash"
      }
    }
  ]
}
