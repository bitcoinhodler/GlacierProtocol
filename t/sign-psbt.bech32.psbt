cHNidP8BAFICAAAAAf8mQGPc1ZGaLa0XwIV/X35+F1tfnafPR/DzSpksx+UsAQAAAAD/////AZcw9AUAAAAAFgAUzmK6QaX8Nd/12XU0AE+YTWqJeX0AAAAAAAEBIPk39AUAAAAAF6kUrVD7JIk1MAjnWwGTAR/x9LdveYyHAQQiACBlyGPHUDMcwD77TayiF+eSXP+H9enKmSdWQjS+DWJ46AEFi1IhA9FN3PtoF/VXlpW7s+s+GFiHvylCsDHm9xY0O4/n6ejiIQKPzUb4YUssvzGAlpaCQqHiK8+22PK23JOcjCfDR7KTeyEDFay1UBIPTNy0YNXEkICrUIykzNje3srCL+6sjNAX1MEhAisGPuLyL54ZgsFA/neGcsaDERp5bczNvUe7ZePWCKmDVK4AAA==
