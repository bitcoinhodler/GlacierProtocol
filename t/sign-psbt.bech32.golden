Are you running this on a computer WITHOUT a network connection of any kind? (y/n)?Have the wireless cards in this computer been physically removed? (y/n)?Are you running on battery power? (y/n)?Are you running on an operating system booted from a USB drive? (y/n)?Is your screen hidden from view of windows, cameras, and other people? (y/n)?Are smartphones and all other nearby devices turned off and in a Faraday bag? (y/n)?Input a filename located in the current directory which contains the PSBT:
0.99891193 BTC in unspent inputs from cold storage address 2N93du8YobdgsHyu3qgBvSyhGUT52utMNeA
0.99889303 BTC going to destination address bcrt1qee3t5sd9ls6alawew56qqnucf44gj7tajk3232
Fee amount: 0.00001890
Confirm? (y/n): 
How many private keys will you be signing this transaction with (at least 2 required to complete transaction)?
#: Key #1: Key #2: 
Is this data correct?
*** WARNING: Incorrect data may lead to loss of funds ***

0.99891193 BTC in unspent inputs from cold storage address 2N93du8YobdgsHyu3qgBvSyhGUT52utMNeA
0.99889303 BTC going to destination address bcrt1qee3t5sd9ls6alawew56qqnucf44gj7tajk3232
Fee amount: 0.00001890

Signing with private keys: 
cQCrT9Ncs9729ao7jbmAWrD9z7tF64s2yKzmD6nkiLAi9sXVZWAn
cP65UeSDZPiTLB6CBwasWv9oJYEjRgQXhswfwcT9HscEKDcEbgy4


Confirm? (y/n): 
Calculating transaction...

Final fee rate: 10.00 satoshis per vbyte

Raw signed transaction (hex):
02000000000101b674c33da8bbd0307e797ef1bb00790e24c87bbbd4365f74f4dbfa491c8c4447010000002322002065c863c750331cc03efb4daca217e7925cff87f5e9ca9927564234be0d6278e8ffffffff019730f40500000000160014ce62ba41a5fc35dff5d97534004f984d6a89797d040047304402203d122d6cb5c2ab9a937ed9a8d75ba97803d2833034a687505623f531211f1fcf022057eb3975c586159f5fb57b7103cbe831fecc357843983689c1c7e73afdbd870401473044022063b978b9c2a41562adf242c660afc8ea5834a04a9a117add258f7c3624b2dbe902207896204b1345094b7a3a17bf4a642bff3f936448cb11ce2a668a56a7c3277d3d018b522103d14ddcfb6817f5579695bbb3eb3e185887bf2942b031e6f716343b8fe7e9e8e221028fcd46f8614b2cbf318096968242a1e22bcfb6d8f2b6dc939c8c27c347b2937b210315acb550120f4cdcb460d5c49080ab508ca4ccd8dedecac22feeac8cd017d4c121022b063ee2f22f9e1982c140fe778672c683111a796dcccdbd47bb65e3d608a98354ae00000000

Transaction fingerprint (md5):
de9b0f53705718486d0c6f4eb996eee3
QR code for transaction written to transaction.png
